### PR TITLE
Initial work on audit_rules_dac_modification templating

### PR DESCRIPTION
**Pull request by rsprudencio**
Added possibility to change delimiter when using csv python module.

I'll add more rules in following commits, just pushed this one right now so the skeleton could be reviewed

### CODE REVIEW

File: shared/templates/template_common.py [hunk 1]
```diff
@@ -30,6 +30,8 @@ def __init__(self, msg):
 
 
 class FilesGenerator(object):
+    delimiter = ','
```
Comment (mpreisler) [inline shared/templates/template_common.py]: what is this used for?

File: shared/templates/create_audit_rules_dac_modification.py [hunk 2]
```diff
@@ -12,6 +12,7 @@
 
 class AuditRulesDacModificationGenerator(FilesGenerator):
     def __init__(self):
+        FilesGenerator.__init__(self)
```
Comment (mpreisler) [inline shared/templates/create_audit_rules_dac_modification.py]: do this with the `super` operator as we discussed before.

**Comment by rsprudencio** [comment]
This is ready for full review

**Comment by mpreisler** [comment]
@rsprudencio I just noticed that the patterns in the CSV are unreadable and extremely repetitive. Could we do better? Let's just put the part that actually changes in there and not the whole pattern.

**Comment by rsprudencio** [comment]
@mpreisler I updated it with a simplified version, but it changes lremovexattr regex which I assume it was wrong at first place. In lremovexattr regex it was checking for both lremovexattr and chmod, I assume chmod was a mistake as it was the only one with 2 attributes.

File: shared/templates/create_audit_rules_dac_modification.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+import re
+
+from template_common import FilesGenerator, UnknownTargetError
+
+
+class AuditRulesDacModificationGenerator(FilesGenerator):
+    def __init__(self):
+        super(AuditRulesDacModificationGenerator, self).__init__()
+        self.delimiter = '&'
```
Comment (mpreisler) [inline shared/templates/create_audit_rules_dac_modification.py]: this whole thing is not needed anymore, I would keep the ability to have different separators, that's a cool feature and we might use it in the future. but in audit rules we don't need it

File: shared/templates/template_common.py [hunk 4]
```diff
@@ -155,7 +157,7 @@ def csv_map(self, filename, language):
         with open(filename, 'r') as csv_file:
             filtered_file = self.filter_out_csv_lines(csv_file, language)
 
-            csv_lines_content = csv.reader(filtered_file)
+            csv_lines_content = csv.reader(filtered_file, delimiter = self.delimiter)
```
Comment (mpreisler) [inline shared/templates/template_common.py]: pep8...

**Comment by mpreisler** [comment]
ACK

**Pull request merged**